tensorfile.py

```diff
@@ -1,20 +1,16 @@
-
 import numpy as np
 from torch.utils.tensorboard import SummaryWriter
 
 loss_list = []
-file_name1 = "loss.txt"
-file1 = open(file_name1, "r")
-for line in file1:
+with open("loss.txt", 'r') as file1:
+    line = file1.readline()
```

Comment:
Did you intend to read only one first line? Or did you want to read all lines? For the latter case, `file1.read()` or `file1.readlines()` can be used.